### DISCUSSION TITLE
Location of themes folder

### DISCUSSION THREAD

**Discussion by nagtegaal** [matt8707/ha-fusion#124] [Q&A]
I would love to work on more themes for this, but I'm unable to find the installed location on my local installation of home assistant (running as an add-on). What folder is this ha-fusion add-on being installed in? I can't seem to find it unfortunately...

**Comment by janiskelemen** [comment]
When developing you won't use fusion in HA itself but start the dev environment locally on your machine. When you follow the development instructions in the README.md you can then launch it locally (it opens a browser window automatically). You will need to add your HA IP address in the .env file. This way it can connection to your HA so you can use your entities. Then you can add themes in the static/themes/ folder.
